==35129== Invalid read of size 8
==35129==    at 0x109F47: estaEnCadena(unsigned int, _rep_cadena*) (cadena.cpp:69)
==35129==    by 0x1097B8: main (principal.cpp:136)
==35129==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==35129== 
==35129== 
==35129== Process terminating with default action of signal 11 (SIGSEGV)
==35129==  Access not within mapped region at address 0x0
==35129==    at 0x109F47: estaEnCadena(unsigned int, _rep_cadena*) (cadena.cpp:69)
==35129==    by 0x1097B8: main (principal.cpp:136)
==35129==  If you believe this happened as a result of a stack
==35129==  overflow in your program's main thread (unlikely but
==35129==  possible), you can try to increase the size of the
==35129==  main thread stack using the --main-stacksize= flag.
==35129==  The main thread stack size used in this run was 8388608.
1>#  implementar primeroEnCadena cadenaSiguiente.
2>#  cadena vacÃ­a.
3>La cantidad es 0.
4>timeout: la orden monitorizada ha volcado un 'core'
Segmentation fault
